src/rpc/evo.cpp

```diff
@@ -611,7 +611,7 @@ static UniValue protx_register_common_wrapper(const JSONRPCRequest& request,
         EnsureWalletIsUnlocked(wallet.get());
     }
 
-    bool isV19active = llmq::utils::IsV19Active(WITH_LOCK(cs_main, return chainman.ActiveChain().Tip();));
+    const bool isV19active{DeploymentActiveAfter(WITH_LOCK(cs_main, return chainman.ActiveChain().Tip();), Params().GetConsensus(), Consensus::DEPLOYMENT_V19)};
```

Comment:
I'm worried this is actually an issue with the locking of cs_main here... Are we sure that we don't need to hold cs_main while DeploymentActiveAfter is called? otherwise the pointer could maybe be invalidated if the block is discarded? not positive @UdjinM6 ?

Reply:
the pointer won't become invalid.

Bitcoin implementation assumes that no CBlockIndex ptr ever becames 0 since they got to any chain. There's no single place in whole implementation that locks `cs_main` for working with any pindex.